README.md

```diff
@@ -3,7 +3,7 @@
 Our blog and tech site has moved to http://multithreaded.stitchfix.com  Please follow us there and
 subscribe to theb log.
```

Comment:
I was hoping *theb log* was our new branding

*Jon Dean*
Principal Engineer
*Stitch Fix | **your partner in personal style*

*technology.stitchfix.com <http://technology.stitchfix.com>*

jon@stitchfix.com
412-596-8348 (Eastern Time)

On Mon, Oct 5, 2015 at 2:11 PM, Geoff Thé <notifications@github.com> wrote:

> In README.md
> <https://github.com/stitchfix/stitchfix.github.io/pull/111#discussion_r41177229>
> :
>
> > @@ -3,7 +3,7 @@
> >  Our blog and tech site has moved to http://multithreaded.stitchfix.com  Please follow us there and
> >  subscribe to theb log.
>
> s/theb log/the blog/ -- not your typo, can we fix while we're here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/stitchfix/stitchfix.github.io/pull/111/files#r41177229>
> .
>